cmd/veneur-emit/main.go

```diff
@@ -363,6 +366,29 @@ func createMetric(span *ssf.SSFSpan, passedFlags map[string]flag.Value, name str
 			span.Metrics = append(span.Metrics, ssf.Timing(name, ended.Sub(start), time.Millisecond, tags))
 		}
 
+		sf, shas := passedFlags["span_starttime"]
+		ef, ehas := passedFlags["span_endtime"]
+		if shas != ehas {
```

Comment:
will this be != if they return different strings on flag parse error? should this be? `if shas != nil || ehas != nil {`

Reply:
It shouldn't be possible to reach an error in parsing here. Because we're relying on the global `flag.FlagSet` (via the call to `flag.Parse()` in our `flags()` func), and that's set to `flag.ExitOnError`, it means that any problems with parsing inputs will cause the whole process to exit way back up there.

The second values here, `shas` and `ehas`, are always going to be booleans - using the two-value assignment form when retrieving from a map means that the second value will be a boolean indicating whether or not there was value in the map for the provided key.